### ISSUE TITLE
field default '' missing if field comment changed

### ISSUE THREAD

**Issue by phoenixg** [Sequel-Ace/Sequel-Ace#1569]
- Sequel Ace Version (including build number):Version 3.5.3 Build 20035
- macOS Version:MacBook Pro (13-inch, M1, 2020)
- Processor Type (Intel/Apple):Apple
- MySQL Version:8
- macOS Localization: zh_CN

**Description**
If change comment of a field which is set `default ''`, the `default ''` will missing

For example:

```
  `my_field` char(20) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '' COMMENT 'my field',
```

Then change the comment, and the field will be:

```
  `my_field` char(20) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL COMMENT 'my field changed',
```

**Comment by Jason-Morcos** [comment]
Can you provide the specific steps to take in the GUI to show this issue? Please include a table create statement and specific set of steps to reproduce this issue.

**Comment by Jason-Morcos** [comment]
Please try the latest beta and confirm it's all good - the fix for this issue is included in the latest 4.0.5+ beta!

https://github.com/Sequel-Ace/Sequel-Ace/releases
